eesp-ikev2.org

```diff
@@ -307,10 +328,11 @@ requirements.
 
 - Protocol ID (1 octet) - MUST be 0. MUST be ignored if not 0.
 - SPI Size (1 octet) - MUST be 0. MUST be ignored if not 0.
-- Notify Status Message Type (2 octets) - set to [TBD1]</t>
-- Minimum number of Sub SAs (2 octets). MUST be greater than 0.
+- Notify Status Message Type (2 octets) - set to [TBD1]
+- Minimum number of Sub SAs (2 octets). MUST be greater than 0,
   If 0 is received, it MUST be interpreted as 1.
```

Comment:
I don't think this is good idea to treat 0 as 1. But let us first synchronize our understanding of Sub SAs :-)